data/json/items/resources/wood.json

```
@@ -119,28 +119,64 @@
     "name": { "str": "plank" },
     "description": "A narrow, thick plank of wood, like a 2 by 4 or similar piece of dimensional lumber.  Makes a decent melee weapon if you're desperate, and can be used for all kinds construction.",
     "category": "spare_parts",
-    "weight": "2600 g",
+    "weight": "2440 g",
     "color": "brown",
     "symbol": "/",
     "material": [ "wood" ],
     "techniques": [ "WBLOCK_1" ],
-    "volume": "4400 ml",
-    "longest_side": "130 cm",
+    "volume": "4130 ml",
+    "longest_side": "122 cm",
     "bashing": 16,
     "to_hit": { "grip": "none", "length": "long", "surface": "any", "balance": "uneven" },
     "price": 1000,
     "price_postapoc": 10,
     "flags": [ "FRAGILE_MELEE", "FIREWOOD" ]
   },
+  {
+    "type": "GENERIC",
+    "id": "plank_short",
+    "name": { "str": "short plank" },
+    "description": "A short 2 by 4 plank of wood.  While less common and smaller than its 4 foot cousin it can still be very useful in carpentry.",
```

```suggestion
    "description": "A short 2 by 2 plank of wood.  While less common and smaller than its 4 foot cousin it can still be very useful in carpentry.",
```
You should probably give different dimensions for a shorter plank than a 2x4.

---

Absolutely not. 2x4 is width and depth, not length. They're still a 2x4